redisinsight/ui/src/pages/vector-search/create-index/hooks/useCreateIndex.ts

```diff
@@ -57,7 +63,7 @@ export const useCreateIndex = (): UseCreateIndexResult => {
         const data = await executeQuery(instanceId, cmd)
 
         // Step 3: Persist results locally so Vector Search history (CommandsView) shows it
-        if (Array.isArray(data) && data.length) {
+        if (envDependentFlag === false && Array.isArray(data) && data.length) {
```

Comment:
We don't want to preserve the commands inside IndexDB always, it should depend on this feature flag, as it's handled in the Workbench.